app/views/timelines/show.html.erb

```diff
@@ -0,0 +1,167 @@
+                  <a href="profile.html" class="thumbnail"><img src=/"images/icon_photo_small.png" alt=""></a>
+                  <a href="profile.html" class="thumbnail"><img src=/"images/icon_photo_small.png" alt=""></a>
+                </div>
+              </div> <!-- end row -->
+
+              <div class="text-center">
+                <a href="#" class="text-center">See More Photos</a>
+              </div>
+
+            </div> <!-- end panel -->
+          </div> <!-- end friends panel-->
+
+
+          <div class="panel panel-default friends">
+            <div class="panel-heading ">
+              <h3 class="panel-title timeline-panel-header">Friends <span class="grey-text">(542)</span></h3>
+            </div>
+            <div class="panel-body">
+              <div class="row">
+                <div class="col-sm-4">
+                  <a href="profile.html" class="thumbnail"><img src="/images/user_silhouette_generic.gif" alt=""></a>
+                  <a href="profile.html" class="thumbnail"><img src="/images/user_silhouette_generic.gif" alt=""></a>
+
+                </div>
+                <div class="col-sm-4">
+                  <a href="profile.html" class="thumbnail"><img src="/images/user_silhouette_generic.gif" alt=""></a>
+                  <a href="profile.html" class="thumbnail"><img src="/images/user_silhouette_generic.gif" alt=""></a>
+
+                </div>
+                <div class="col-sm-4">
+                  <a href="profile.html" class="thumbnail"><img src="/images/user_silhouette_generic.gif" alt=""></a>
+                  <a href="profile.html" class="thumbnail"><img src="/images/user_silhouette_generic.gif" alt=""></a>
+
+                </div>
+              </div> <!-- end row -->
+              <div class="text-center">
+                <a href="#" class="text-center">See More Friends</a>
+              </div>
+
+            </div>
+          </div> <!-- end friends panel-->
+
+
+        </div> <!--col-md -4 -->
+
+
+      dldmlmdlmdlm
+      <%= current_user.id %> current user
+      <%= @user.id %>  at user
+        <div class="col-md-8">
+          <div class="panel panel-default">
+            <div class="panel-heading">
+              <h3 class="panel-title timeline-panel-header">Post</h3>
+            </div>
+
+            <div class="panel-body">
+              <%= form_for [ @post] do |f|%>            
+                <div class="form-group">
+                  <%= f.text_area :body, 'Make a wall post',class: 'form-control' %>
+                </div>
+                  
+              
+
+
+
+            </div>
+
+            <div class="panel-footer clearfix">
+               <%= f.submit 'Post', class: 'pull-right btn btn-sm btn-primary' %>
+             
+            </div>
+             <% end %> 
+          </div> <!-- end panel default -->
+
+          <% @posts.each do |post| %>
```

Comment:
This likely can be refactored into a partial.

You may even be able to use the shortcut <%= render @posts %>